src/Rector/Deprecation/FileUnmanagedCopyRector.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use DrupalRector\Utility\TraitsByClassHelperTrait;
+use PhpParser\Node;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
```

Comment:
@mohsin-saeed, with the update to Rector, all of the `Rector\` class names need to be changed to `Rector\Core\`.

Sorry for moving things from underneath you. :(